samples/subsys/usb/cdc_acm/prj.conf

```diff
@@ -6,7 +6,7 @@ CONFIG_USB_DEVICE_PRODUCT="Zephyr CDC ACM sample"
 CONFIG_LOG=y
 CONFIG_USB_DRIVER_LOG_LEVEL_ERR=y
 CONFIG_USB_CDC_ACM=y
-CONFIG_USB_DEVICE_LOG_LEVEL_ERR=y
+CONFIG_USB_CDC_ACM_LOG_LEVEL_INF=y
```

Comment:
Those are different logs, we have `USB_DEVICE_LOG_LEVEL` defined for other samples, I do not see the reason you remove it.

It is OK to add `USB_CDC_ACM_LOG_LEVEL_INF` for this sample, please also add the same for `cdc_acm_composite`

Reply:
> Those are different logs, we have `USB_DEVICE_LOG_LEVEL` defined for other samples, I do not see the reason you remove it.

Ah.  The names were too close, didn't notice that one was for `DRIVER` and one for `DEVICE`--only that I wasn't getting the log messages I wanted.  Good call.

In that case I misunderstood the role of the existing setting, adding `CDC_ACM` would be a new thing and I don't care to go through all the other samples and add it there.  I'll drop the PR.